util/logging.c

```diff
@@ -167,3 +176,103 @@ int nvme_submit_passthru64(int fd, unsigned long ioctl_cmd,
 
 	return err;
 }
+
+static void nvme_show_req_admin(const struct nvme_mi_admin_req_hdr *hdr, size_t hdr_len,
+				const void *data, size_t data_len)
+{
+	struct nvme_passthru_cmd cmd = {
+		.opcode = hdr->opcode,
+		.flags = hdr->flags,
+		.nsid = le32_to_cpu(hdr->cdw1),
+		.cdw2 = le32_to_cpu(hdr->cdw2),
+		.cdw3 = le32_to_cpu(hdr->cdw3),
+		.addr = (uint64_t)(uintptr_t)data,
+		.data_len = data_len,
+		.cdw10 = le32_to_cpu(hdr->cdw10),
+		.cdw11 = le32_to_cpu(hdr->cdw11),
+		.cdw12 = le32_to_cpu(hdr->cdw12),
+		.cdw13 = le32_to_cpu(hdr->cdw13),
+		.cdw14 = le32_to_cpu(hdr->cdw14),
+		.cdw15 = le32_to_cpu(hdr->cdw15),
+	};
+
+	nvme_show_common(&cmd);
```

Comment:
I think we would want include the `dofst` / `dlen` here too, otherwise the data may not make much sense.